src/services/perimeters.ts

```diff
@@ -15,38 +14,20 @@ import {
 } from 'utils/graphUtils'
 import { getApiResponseResources } from 'utils/apiHelpers'
 
+import { CONTEXT } from '../constants'
+import { getLastEncounter } from './myPatients'
 import fakeGroup from '../data/fakeData/group'
 import fakeFacetDeceased from '../data/fakeData/facet-deceased'
 import fakeFacetAgeMonth from '../data/fakeData/facet-age-month'
 import fakeFacetClassSimple from '../data/fakeData/facet-class-simple'
 import fakeFacetStartDateFacet from '../data/fakeData/facet-start-date-facet'
 import fakePatients from '../data/fakeData/patients'
+import { FHIR_API_Response, CohortData, ScopeTreeRow } from 'types'
 
-export const getServices = async (id: string) => {
-  const [respOrganizations, respHealthcareServices] = await Promise.all([
-    api.get<FHIR_API_Response<IOrganization>>(`/Organization?_id=${id}${API_RESOURCE_TAG}`),
-    api.get<FHIR_API_Response<IHealthcareService>>(`/HealthcareService?_id=${id}${API_RESOURCE_TAG}`)
-  ])
-
-  const organizations = getApiResponseResources(respOrganizations)
-  const healthcareServices = getApiResponseResources(respHealthcareServices)
-  if (!organizations || !healthcareServices) {
-    return []
-  }
-  let services: IHealthcareService[] = [...healthcareServices]
-
-  for (const orga of organizations) {
-    if (orga.id) {
-      const impliedServiceResp = await api.get<FHIR_API_Response<IHealthcareService>>(
-        `/HealthcareService?organization=${orga.id}${API_RESOURCE_TAG}`
-      )
-      const impliedServices = getApiResponseResources(impliedServiceResp)
-      if (impliedServices) {
-        services = [...services, ...impliedServices]
-      }
-    }
-  }
-  return services
+export const getOrganizations = async (ids?: string): Promise<IOrganization[]> => {
```

Comment:
how about accepting an array of ids and then `join(',')`? Would not this be safer ?

Reply:
I agree, but it turns out changing it this way has lots of border effects.. :(